server/views/referrals/new/courseParticipations/index.njk

```diff
@@ -52,23 +31,47 @@
             "data-testid": "success-banner"
           }
         }) }}
-      {% endif %}
+      {% else %}
```

Comment:
I think when we met to discuss these changes we agreed to include the inset text and paragraph regardless of whether the 'success' banner was there, which might simplify some of the integration testing?

(This one was like this in the earlier designs, so I think it's just not been updated after our chat - again, I think we can keep it simple)